doc/classes/OS.xml

```diff
@@ -417,6 +424,7 @@
 			<return type="bool">
 			</return>
 			<description>
+				Returns whether or not the device has a touchscreen or emulates one.
```

Comment:
Per writing guidelines, boolean methods and functions should prefer "[code]true[/code]" construction over "whether".

    Returns [code]true[/code] if the device has a touchscreen or emulates one.